doc/guide/install.rst

```diff
@@ -122,12 +135,13 @@ For advanced Python users or developers of CLIMADA, we recommed cloning the CLIM
 
    Both commands will probably require administrator rights, which can be enabled by prepending ``sudo``.
 
-#. Create a **workspace directory**.
+#. Create a folder for your source code.
```

Comment:
```suggestion
#. Create a folder where you will save the code that you will write. 
```
I think the word `source code` might frighten some novice users.

Reply:
More simple?
```suggestion
#. Create a folder for your code.
```

Reply:
Agreed, `create a folder for your code` is better.